ArcadiaParties.CQRS/Handlers/GetAssistantEmployeeHandler.cs

```diff
@@ -0,0 +1,46 @@
+﻿using ArcadiaParties.CQRS.Queries;
+using ArcadiaParties.Data.Abstractions.DTOs;
+using ArcadiaParties.Data.Abstractions.Repositories;
+using MediatR;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ArcadiaParties.CQRS.Handlers
+{
+    class GetAssistantEmployeeHandler : IRequestHandler<GetAssistantEmployeeQuery, AssistantEmployeeDTO>
+    {
+        private readonly IAssistantTokenRepository _tokenRepository;
+        private readonly IHttpClientFactory _clientFactory;
+        private readonly IMediator _mediator;
+
+        public GetAssistantEmployeeHandler(IAssistantTokenRepository tokenRepository, IMediator mediator, IHttpClientFactory clientFactory)
+        {
+            _tokenRepository = tokenRepository;
+            _mediator = mediator;
+            _clientFactory = clientFactory;
+        }
+
+        const string requestEmployee = "https://assistant.arcadia.spb.ru/api/employees/";
```

Comment:
Access modifier is missed. Constants should be places at the top of the class.

Reply:
Fixed